### ISSUE TITLE
Code to notify user when room is full is in available but it does not work!

### ISSUE THREAD

**Issue by iambenkay** [ianramzy/decentralized-video-chat#30]
I noticed that the following line quotes here should limit users once the room has 2 participants but it does not work.
https://github.com/ianramzy/decentralized-video-chat/blob/6101706fa742015e6b9d94b93c58dbaf2fab3eeb/server.js#L91

**Comment by Chaphasilor** [comment]
Could you elaborate on this? What exactly is not working here?

**Comment by ianramzy** [comment]
The problem here is that once two people are connected in a room the server
disconnects, essentially making the room name available again for more to
join. To fix this you would need to keep a persistent connection to the
server even after calls have connected.

**Comment by Chaphasilor** [comment]
How about instead of a persistant connection, we make the clients, possibly only the 'host' (or innitiator of the call) ping the server every few minutes?
The server just keeps a list of active rooms and removes the rooms after not getting a ping for a few minutes?

Or we could make both clients ping the server and the server hashes each client's IP and links it to the room, so than once one client disconnects from a room, another client can join. We'd need to increase the ping interval, but that shouldn't be an issue.

This should guarantee anonymity for the clients and prevent this issue from happening?

**Comment by ianramzy** [comment]
Yes that would work, and be a simple solution. Another option is to try and
get group calling working.

**Comment by Chaphasilor** [comment]
Wouldn't we still need limit the size of the call? Depending on the topology we use, we can't support a call with 8 people or more...
